.github/workflows/pinot_tests.yml

```diff
@@ -78,19 +78,9 @@ jobs:
       fail-fast: false
       matrix:
         testset: [ 1, 2 ]
-        java: [ 11, 21 ]
+        java: [ 11, 21, 24 ]
```

Comment:
I don't think https://github.com/apache/datasketches-java works with Java 24 yet.

Reply:
Oh, that is sad. I'll try to take a look on how to solve that (or ignore these test). It seems hadoop and spark have issues with Java 24 as well.

Reply:
BTW, I've just found that data https://github.com/apache/datasketches-memory will require Java >= 21 (and --enable-preview in 21)